repos/system_upgrade/common/actors/scan_default_initramfs/actor.py

```diff
@@ -0,0 +1,20 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import scan_default_initramfs as scan_default_initramfs_lib
+from leapp.models import DefaultInitramfsInfo, DefaultSourceBootEntry
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class ScanSourceBootEntry(Actor):
```

Comment:
Seems like a copy paste err
```suggestion
class ScanDefaultInitramfs(Actor):
```